calico-cloud/_includes/content/_kibana-indices.mdx

```diff
@@ -0,0 +1,15 @@
+An EndpointPort associates a name with a particular TCP/UDP/SCTP port of the endpoint, allowing it to
```

Comment:
This file looks to be a new include file but it does not appear to be used anywhere.

Reply:
Removed.